specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-01-30-preview/definitions/Projects.json

```diff
@@ -0,0 +1,163 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Database Migration Service Resource Provider",
+    "version": "2022-01-30-preview"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "Project": {
+      "type": "object",
+      "description": "A project resource",
+      "properties": {
+        "properties": {
+          "description": "Project properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ProjectProperties"
+        },
+        "systemData": {
+          "$ref": "../sqlmigration.json#/definitions/SystemData",
+          "description": "Metadata pertaining to creation and last modification of the resource.",
+          "readOnly": true
+        },
+        "etag": {
+          "type": "string",
+          "description": "HTTP strong entity tag value. This is ignored if submitted."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../sqlmigration.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ProjectList": {
+      "type": "object",
+      "description": "OData page of project resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of projects",
+          "items": {
+            "$ref": "#/definitions/Project"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to load the next page of projects"
+        }
+      }
+    },
+    "ProjectProperties": {
+      "type": "object",
+      "description": "Project-specific properties",
+      "properties": {
+        "sourcePlatform": {
+          "description": "Source platform for the project",
+          "$ref": "#/definitions/ProjectSourcePlatform"
+        },
+        "azureAuthenticationInfo": {
+          "description": "Field that defines the Azure active directory application info, used to connect to the target Azure resource",
+          "type": "object"
```

Comment:
>object [](http://example.com/codeflow?start=19&length=6)

Why is this an anonymous object with no schema?

Reply:
added now